Streetcode/Streetcode.BLL/MediatR/Media/Image/GetByStreetcodeId/GetImageByStreetcodeIdQueryHandler.cs

```diff
@@ -20,7 +20,7 @@ public GetImageByStreetcodeIdQueryHandler(IRepositoryWrapper repositoryWrapper,
 
         public async Task<Result<IEnumerable<ImageDTO>>> Handle(GetImageByStreetcodeIdQuery request, CancellationToken cancellationToken)
         {
-            var image = await _repositoryWrapper.ImageRepository.GetSingleOrDefaultAsync(f => f.Streetcodes.Any(s => s.Id == request.streetcodeId));
+            var image = await _repositoryWrapper.ImageRepository.GetAllAsync(f => f.Streetcodes.Any(s => s.Id == request.streetcodeId));
             if (image == null)
```

Comment:
- Please insert a blank line above;
- Please use `__value is null__` instead of `__value == null__`.